mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.java

```diff
@@ -1164,4 +1188,14 @@ private void manageShortcut(boolean visible, String id, String action,
                     getString(disableMessage));
         }
     }
+
+    private void setVoiceWidgetComponentEnabledSetting(Class<?> component,
+                                                       boolean isSpeechRecognizerAvailable) {
+        ComponentName voiceWidget = new ComponentName(BuildConfig.APPLICATION_ID,
+                component.getCanonicalName());
+        PackageManager pm = getApplicationContext().getPackageManager();
```

Comment:
Can simplify to `getPackageManager()` (routing via application context seems unneeded)